packages/main/cypress/specs/Menu.cy.ts

```diff
@@ -205,6 +205,43 @@ describe("Menu interaction", () => {
 			.should("be.focused");
 	});
 
+	it("Set focus on first item", () => {
+		cy.mount(html`<ui5-button id="btnOpen">Open Menu</ui5-button>
+		<ui5-menu id="menu" loading loading-delay="100">
+		</ui5-menu>`);
+
+		cy.get("#btnOpen")
+			.as("button")
+			.realClick();
+
+		cy.get("[ui5-menu]")
+			.as("menu").then($menu => {
+				$menu.get(0).addEventListener("ui5-before-open", () => {
+					setTimeout(() => {
+						$menu.removeAttr("loading");
```

Comment:
you can set this via property set, no need to changed via set/remove attribute

Reply:
one time you work with real dom reference and one time you work with jQuery object.

```
		const menu = $menu.get(0);

				menu.addEventListener("ui5-before-open", () => {
					setTimeout(() => {
						menu.loading = false;
						menu.loadingDelay = 0;

						const oneNode = document.createElement("ui5-menu-item") as MenuItem;
						oneNode.text = "Open from Amazon Cloud";

						const twoNode = document.createElement("ui5-menu-item") as MenuItem;
						twoNode.text = "Open from Google Cloud";

						menu.append(oneNode, twoNode);
						menu.focus();
					}, 1000);
				});
```